Formatting changes to templates (#1667)
GoogleCloudPlatform/DataflowTemplates: v2/file-format-conversion/README_File_Format_Conversion.md

### Optional parameters

* **containsHeaders** : Input CSV files contain a header record (true/false). Only required if reading CSV files. Defaults to: false.
* **deadletterTable** : Messages failed to reach the target for all kind of reasons (e.g., mismatched schema, malformed json) are written to this table. (Example: your-project:your-dataset.your-table-name).
* **delimiter** : The column delimiter of the input text files. Default: `,` (Example: ,).
* **csvFormat** : CSV format specification to use for parsing records. Default is: `Default`. See https://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVFormat.html for more details. Must match format names exactly found at: https://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVFormat.Predefined.html.
* **jsonSchemaPath** : The path to the JSON schema. Defaults to: `null`. (Example: gs://path/to/schema).
* **largeNumFiles** : Set to true if number of files is in the tens of thousands. Defaults to: `false`.
* **csvFileEncoding** : The CSV file character encoding format. Allowed Values are `US-ASCII`, `ISO-8859-1`, `UTF-8`, and `UTF-16`. Defaults to: UTF-8.

> * **logDetailedCsvConversionErrors** : Set to `true` to enable detailed error logging when CSV parsing fails. Note that this may expose sensitive data in the logs (e.g., if the CSV file contains passwords). Default: `false`.

Reviewer: ```suggestion
* **logDetailedCsvConversionErrors** : Set to `true` to enable detailed error logging when CSV parsing fails. This setting might expose sensitive data in the logs, such as password contained in the CSV file. Defaults to `false`.
```
